thread_pthread.c

```diff
@@ -101,6 +101,84 @@
 #  endif
 #endif
 
+static rb_thread_hook_t * rb_thread_hooks = NULL;
+static pthread_rwlock_t rb_thread_hooks_rw_lock = PTHREAD_RWLOCK_INITIALIZER;
+
+rb_thread_hook_t *
+rb_thread_event_new(rb_thread_callback callback, rb_event_flag_t event) {
```

Comment:
could you rename `event` to `internal_event`?

Reply:
Done.